libs/storybook/src/lib/parameters.tsx

```diff
@@ -5,23 +5,23 @@ import {possibleTypes} from '@wepublish/website/api'
 
 const defaultOptions: DefaultOptions = {
   watchQuery: {
-    fetchPolicy: 'no-cache',
```

Comment:
no-cache means it never uses the cache, network-only means the initial data is from the network and then uses the cache.

We update the cache, to test this I switched to network-only which makes the most sense to be begin with in my opinion.